manifests/init.pp

```diff
@@ -64,6 +68,9 @@
   $enable_ostree = $katello::params::enable_ostree,
   $candlepin_event_queue = $katello_devel::params::candlepin_event_queue,
 
+  $pulp_db_username = $katello_devel::params::pulp_db_username,
+  $pulp_db_password = $katello_devel::params::pulp_db_password,
```

Comment:
I think we have the same pattern for some other parameters as well so I'd either add them all (and remove the inheritance) or leave it as it is now.